types/main.d.ts

```diff
@@ -628,3 +628,75 @@ declare module "pdfjs" {
     export type TextAlignment = "left" | "right" | "center" | "justify";
     export type ImageAlignment = "left" | "right" | "center";
 }
+
+declare type Font = import("pdfjs").Font;
+
+declare module "pdfjs/font/Courier-Bold" {
```

Comment:
Hey @rkusa thanks so much for this.
It did work as expected, yes.
Although, I am not sure this can be helpful but you could minimize lines of code by just declaring:

```ts
declare module "pdfjs/font/*" {
    const font: Font;
    export = font;
}
```
instead of explicitly declaring every font?

Reply:
Thanks for the review.
I have though of that, but it allows any fonts including those that not exists.
And on some IDE like VS Code, it allows the autocompletion when importing the font.

Reply:
That makes a lot of sense ;)